stellargraph/utils/loss.py

```diff
@@ -14,27 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
+import warnings as warnings
 
-from ..core.experimental import experimental
+warnings.warn(
+    "'stellargraph.utils.loss' has been moved to 'stellargraph.losses'",
+    DeprecationWarning,
+)
```

Comment:
Is this still needed with the Wrapper in `stellargraph/utils/__init__.py`?

Reply:
Yes; there's two code paths here:

1. `import stellargraph.utils` followed by `utils.foo`: this takes the `__getattr__` route
2. a direct import of a submodule `import stellargraph.utils.foo`: I believe this just does a file system traversal and opens/executes the given file (i.e. `stellargraph/utils/foo.py` or `stellargraph/utils/foo/__init__.py`) to create the submodule directly (rather than getting attributes of parent modules), and so needs handling in each file individually

(For this specific file, it isn't reexported in `utils/__init__.py`, and so `utils.loss` in 1 doesn't actually work, and only 2 works.)